keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/Jooq.kt

```diff
@@ -0,0 +1,9 @@
+package com.netflix.spinnaker.keel.sql
+
+import org.jooq.DSLContext
+import org.jooq.impl.DSL
+
+internal fun <R> DSLContext.inTransaction(fn: DSLContext.() -> R): R =
```

Comment:
Pulled out of `SqlResourceRepositoryTests` and just changed to possibly return a value if the closure does.